pandas/util/testing.py

```diff
@@ -2263,37 +2263,6 @@ def wrapper(*args, **kwargs):
 with_connectivity_check = network
 
 
-class SimpleMock(object):
```

Comment:
Looks like SimpleMock is used in `stdin_encoding` below. Can you see if `stdin_encoding` is used anywhere, and maybe remove it too?

Reply:
@TomAugspurger Sure, will do